### DISCUSSION TITLE
멀티플레이 샘플 프로젝트에서 [1.1.5]에서 [1.1.7]로 업데이트시 문제발생

### DISCUSSION THREAD

**Discussion by TaejunPark** [naverz/zepeto-studio-kor#617] [Issue Report (Legacy)]
업데이트 했더니
001_Desert 에서
기존에 있던 WorldMultiplay 오브젝트의 스크립트 연결이 끊어지고, 컴포넌트로 ZepetoWorldMultiplay 스크립트가 뜨질 않습니다
<img width="420" alt="스크린샷 2022-04-14 오후 1 19 41" src="https://user-images.githubusercontent.com/3603377/163312967-efdf8ef3-69eb-4e58-9d2c-ec34e90fb259.png">

샘플을 수정해서 시작한 제 프로젝트를 다시 돌아가게 하고 싶은데, 어떻게 하면 될까요 ㅠ

*패키지 업데이트 안내 페이지에  백업 권고가 있었으면 합니다,, 너무 페이지 분위기가 평온해서  저도모르게 맨날 하던걸 안했네요..ㅜ

**Comment by minjujuu** [comment]
한 번 상단 메뉴 Assets - Reimport All 해보시면 나타날 것 같습니다!

**Reply by TaejunPark** [reply]
감사합니다  잘되네영ㅎ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, @minjujuu 님께서 알려주신 방법대로 진행 부탁 드립니다.
월드 버전 업데이트시 백업 안내창을 띄우는 방법은 개발팀과 논의해보겠습니다.

**Reply by TaejunPark** [reply]
그냥  혼자 꽁해서 징징대봤습니다 ㅈㅅ합니다
